src/au3cloud/internal/clouduiactions.cpp

```
@@ -0,0 +1,43 @@
+                       au::context::CTX_ANY,
+                       muse::TranslatableString("action", "View on audio.com"),
+                       muse::TranslatableString("action", "View on audio.com")
+                       )
+};
+
+const muse::ui::UiActionList& CloudUiActions::actionsList() const
+{
+    return m_actions;
+}
+
+bool CloudUiActions::actionEnabled(const muse::ui::UiAction&) const
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)